store/store.go

```diff
@@ -199,7 +199,7 @@ func (bs *BlockStore) SaveBlock(block *types.Block, blockParts *types.PartSet, s
 	bs.db.Set(calcBlockHashKey(hash), []byte(fmt.Sprintf("%d", height)))
 
 	// Save block parts
-	for i := 0; i < blockParts.Total(); i++ {
+	for i := 0; int32(i) < blockParts.Total(); i++ {
```

Comment:
Let's be consistent about which side of this check we cast - I believe the standard is to use the non-bit-specified type (that is, just `int`).

Reply:
Err, to use the non-bit-specified type when doing comparisons _in for loops_.

Reply:
made the change thank you for the pointer